Islandora/islandora_internet_archive_bookreader
# Setting the Default Page View Does not Work

**ISLANDORA-2506**: (https://jira.lyrasis.org/browse/ISLANDORA-2506)

* [Internet Archive Book reader Display Options|https://openlibrary.org/dev/docs/bookurls#options
* [Book reader display options in Git|https://github.com/internetarchive/bookreader/blob/2.0.2/BookReader/BookReader.js#L4946-L4952]

# What does this Pull Request do?
Addresses the bug where selecting a default page view does not work.

# What's new?
This work overrides the bookreader's [paramsFromFragment function|https://github.com/internetarchive/bookreader/blob/2.0.2/BookReader/BookReader.js#L4918], to correct the out of scope javascript variables. Inclusive of this, it sets the bookreader defaults variable.

# How should this be tested?
* Create a book object with multiple pages
* Chose a default page view in the IA book viewer configuration (_<your_site>/admin/islandora/islandora_viewers/internet_archive_bookreader_).
* Save the configuration.
* Navigate to your book, and verify your default page view is shown.


# Additional Notes:
Without this bug fix, the viewer also ignores the defaults due to the out of scope javascript variables (this.constMode1up, this.constMode2up, and this.constModeThumb). As a result, it will choose a page view based on window size.

# Interested parties
@Islandora/7-x-1-x-committers
@patdunlavey
